test/python/qdmi/qiskit/test_qdmi_qiskit_gate_support.py

```
@@ -0,0 +1,491 @@
+    # If not provided, a warning should be issued (tested separately)
+    if "measure" in backend._capabilities.operations:  # noqa: SLF001
+        assert "measure" in backend.target.operation_names
+
+
+def test_backend_target_qubit_count_matches_device() -> None:
+    """Test that backend target qubit count is based on device capabilities."""
+    backend = QiskitBackend()
+
+    # Target qubit count should be at least the device num_qubits
+    # (may be higher if zones are included in the target)
+    assert backend.target.num_qubits >= backend._capabilities.num_qubits  # noqa: SLF001
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)